components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -462,6 +463,19 @@ - (UIView *)viewForItem:(UITabBarItem *)item {
   return _itemViews[itemIndex];
 }
 
+- (UITabBarItem *)tabBarItemForPoint:(CGPoint)point {
+  for (NSUInteger i = 0; i < self.itemViews.count; i++) {
+    UIView *itemView = [self.itemViews objectAtIndex:i];
+    BOOL isPointInView = CGRectContainsPoint(itemView.frame, point);
+    BOOL isInItemsBounds = i < self.items.count;
+    if (isPointInView && isInItemsBounds) {
+      return [self.items objectAtIndex:i];
```

Comment:
Tip: Please use modern array syntax if you can:

```objc
return self.items[i];
```

Reply:
Done.